custom_model_runner/datarobot_drum/resource/predict_mixin.py

```diff
@@ -95,7 +97,11 @@ def _check_mimetype_support(self, mimetype):
     def _predict(self, logger=None):
         response_status = HTTP_200_OK
         try:
+
             binary_data, mimetype, charset = self._fetch_data_from_request("X", logger=logger)
+            sparse_data, _, _ = self._fetch_data_from_request(
+                "X.colnames", logger=logger, optional=True
```

Comment:
I suggest to move `X.colnames` into a constant.... 
Would be nice if you could also move `X` into a constant